controls/V-72225.rb

```diff
@@ -1,179 +1,210 @@
+  tag "fix_id": "F-78579r4_fix"
+
+  #When Banner is commented, not found, disabled, or the specified file does not exist, this is a finding.
+  banner_files = [sshd_config.banner].flatten
+
+  banner_files.each do |banner_file|
+
+    #Banner property is commented out.
+    describe "The SSHD Banner is not set" do
+      subject { banner_file.nil? }
+      it { should be false }
+    end if banner_file.nil?
+
+    #Banner property is set to "none"
+    describe "The SSHD Banner is disabled" do
+      subject { banner_file.match(/none/i).nil? }
+      it { should be true }
+    end if !banner_file.nil? && !banner_file.match(/none/i).nil?
+
+    #Banner property provides a path to a file, however, it does not exist.
+    describe "The SSHD Banner is set, but, the file does not exist" do
+      subject { file(banner_file).exist? }
+      it { should be true }
+    end if !banner_file.nil? && banner_file.match(/none/i).nil? && !file(banner_file).exist?
+
+    #Banner property provides a path to a file and it exists.
+    describe.one do
+      banner = file(banner_file).content.gsub(%r{[\r\n\s]}, '')
+      clean_banner = BANNER_MESSAGE_TEXT_RAL.gsub(%r{[\r\n\s]}, '')
+      clean_banner_limited = BANNER_MESSAGE_TEXT_RAL_LIMITED.gsub(%r{[\r\n\s]}, '')
+
+      describe "The SSHD Banner is set to the standard banner and has the correct text" do
+        subject { banner }
+        it { should cmp clean_banner }
+      end
+
+      describe "The SSHD Banner is set to the standard limited banner and has the correct text" do
+        subject { banner }
+        it { should cmp clean_banner_limited }
+      end
+    end if !banner_file.nil? && banner_file.match(/none/i).nil? && file(banner_file).exist?
```

Comment:
For completeness, there should probably be a test added that connects to SSH and reads the banner.

This can be added as a future enhancement though.